config/projects.yml

```
@@ -549,14 +549,63 @@ fuzzing:
       cloud: gcp
       minCapacity: 0
       maxCapacity: 10
+    decision:
+      owner: fuzzing+taskcluster@mozilla.com
+      emailOnError: false
+      imageset: docker-worker
+      cloud: gcp
+      minCapacity: 0
+      maxCapacity: 10
+  hooks:
+    decision:
+      description: Hook to trigger the Fuzzing decision task
+      owner: fuzzing+taskcluster@mozilla.com
+      emailOnError: true
+      schedule:
+        - "0 0 */1 * * *"
+      task:
+        provisionerId: proj-fuzzing
+        workerType: decision
+        payload:
+          image: python:3.7-slim
+          features:
+            taskclusterProxy: true
+          environment:
+            DEPLOY_KEY: project/fuzzing/deploy-fuzzing-tc-config
+            TASKCLUSTER_SECRET: project/fuzzing/decision
+          command:
+            - /bin/bash
+            - -cxe
+            - |
+              apt-get update
+              apt-get install -qq openssh-client curl git > /dev/null
+              mkdir ~/.ssh
+              ssh-keyscan github.com > ~/.ssh/known_hosts
+              cd ~/.ssh
+              curl -L http://taskcluster/api/secrets/v1/secret/$DEPLOY_KEY | python -c 'import json, sys; a = json.load(sys.stdin); open("id_ed25519", "w").write(a["secret"]["key"])'
```

This works, but the following might be better (in case taskcluster proxy url changes at some point).

```suggestion
              curl -L "${TASKCLUSTER_PROXY_URL}/api/secrets/v1/secret/${DEPLOY_KEY}" | python -c 'import json, sys; a = json.load(sys.stdin); open("id_ed25519", "w").write(a["secret"]["key"])'
```

(see [`TASKCLUSTER_PROXY_URL` docs](https://community-tc.services.mozilla.com/docs/manual/design/env-vars#taskcluster_proxy_url))